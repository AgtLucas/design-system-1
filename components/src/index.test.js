import * as AllComponents from './';

describe('All Components', () => {
  it('matches snapshot', () => {
    expect(AllComponents).toBeDefined();
    expect(AllComponents).toMatchSnapshot();
  });
});
