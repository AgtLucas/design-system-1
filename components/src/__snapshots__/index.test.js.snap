// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApprovalButton matches snapshot 1`] = `[Function]`;

exports[`Button matches snapshot 1`] = `[Function]`;

exports[`DangerButton matches snapshot 1`] = `[Function]`;

exports[`LinkButton matches snapshot 1`] = `[Function]`;
